src/loops/programs_with_repetition/easy/q4/MainTest.java

```diff
@@ -0,0 +1,107 @@
+package loops.programs_with_repetition.easy.q4;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import global.BaseTest;
+import global.exceptions.InvalidClauseException;
+import global.tools.TestOption;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.PlaceHolder;
+import global.variables.clauses.StringLiteral;
+
+public class MainTest extends BaseTest {
+  // Java
+  public Clause[] testSentence() {
+    TestOption.isInputTest = true;
+    TestOption.defaultInput = "2";
+    return new Clause[] {
+        new StringLiteral("Enter an integer\\: "),
+        new NewLine(),
+        new PlaceHolder(),
+    };
+  }
+
+  public void runMain() {
+    ItsAlwaysPrimeTime.main(new String[0]);
+  }
+
+  static Stream<Arguments> allPrimeInputs() {
+    int[] allPrimesUnder50 = { 2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47 };
+    Stream<Arguments> allPrimeInputs = Stream.of();
+    for (int primeNumber : allPrimesUnder50) {
+      allPrimeInputs = Stream.concat(allPrimeInputs, Stream.of(Arguments.of(primeNumber)));
+    }
+    return allPrimeInputs;
+  }
+
+  private static Stream<Arguments> repetitiveHardcodedInputs() {
+    return Stream.of(
```

Comment:
There should also be a prime number in one of these samples that is greater than 50 so we know that does not slip through the program

Reply:
If they forget to check that boundary.